dashboard/static/styles.css

```
@@ -1166,6 +1166,61 @@ td { font-size: 13px; }
+  display: flex;
+  gap: 6px;
+  flex-wrap: wrap;
+}
+.ot-tab-btn {
+  display: inline-flex;
+  align-items: center;
+  gap: 7px;
+  background: rgba(255, 255, 255, 0.04);
+  border: 1px solid var(--border-subtle);
+  color: var(--text-secondary);
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)